src/init.cc

```diff
@@ -2415,6 +2426,12 @@ static ncclResult_t commCleanup(ncclComm_t comm) {
 
 NCCL_API(ncclResult_t, ncclCommFinalize, ncclComm_t comm);
 ncclResult_t ncclCommFinalize_impl(ncclComm_t comm) {
+  CollectiveCall finalizeCall;
+  finalizeCall.coll = rrCommFinalize;
+  finalizeCall.comm = comm;
+  finalizeCall.name = "CommFinalize";
+  NCCLCHECK(Replayer::rcclReplayerRecord(finalizeCall));
```

Comment:
This is just an example of what's valid. Others can decide if this is readable or desirable enough.
```suggestion
  NCCLCHECK(Replayer::rcclReplayerRecord({ .coll=rrCommFinalize, .comm=comm, .name="CommFinalize"}));
```